core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -2806,6 +2810,70 @@ private static Integer getJsonDepth(Object o) {
     return depth;
   }
 
+  private static String jsonPairsModify(Object jsonDoc, String type, Object... kvs) {
+    assert kvs.length % 2 == 0;
```

Comment:
Please help me understand if there were challenges using JSON Path. I'm sorry if it too obvious. https://stackoverflow.com/questions/27244431/how-to-change-values-in-a-json-file-using-xpath-jsonpath-in-java

Reply:
well，I will change it later.Thank you very much.

Reply:
hi @ritesh-kapoor Thank you very much. I have seen that jsonpath source code has modified this PR.